### ISSUE TITLE
[BUG] scrollSeekConfiguration with reversed list: flickering

### ISSUE THREAD

**Issue by eliasws** [petyosi/react-virtuoso#410]
**Describe the bug**
When using scrollSeekConfiguration with a reversed list the scrollSeekConfiguration.onEnter callback is called directly on the first render with a high velocity (probably height of the list) which causes the ScrollSeekPlaceholder to be shown. It seems that this was introduced in version 0.21.0-alpha.1. 

**Reproduction**
https://codesandbox.io/s/react-virtuoso-scrolling-issue-forked-xis50?file=/src/App.js

**To Reproduce**
1. Reload the page
2. Check console, scrollSeekConfiguration.onEnter gets called with a velocity of 1446
3. Items render -> then the ScrollSeekPlaceholder gets rendered for split second -> Items render

**Expected behaviour**
scrollSeekConfiguration.onEnter should not be called on first render without actual scrolling

**Screenshots**
-

**Desktop (please complete the following information):**
 - MacOS, Android
 - chrome, chrome mobile

**Additional context**
isScrolling gets also called on first render.

**Comment by petyosi** [comment]
Confirming that this is a bug, The [scrollSeek system](https://github.com/petyosi/react-virtuoso/blob/master/src/scrollSeekSystem.ts) should wait for the didMount observable.

**Comment by eliasws** [comment]
I tried to implement it and create a PR. At least in this case didMount seems to be always true (so would not have any effect):

` pipe(
        scrollVelocity,
        withLatestFrom(scrollSeekConfiguration, isSeeking, rangeChanged, didMount),
        filter(([_, config]) => !!config),
        map(([speed, config, isSeeking, range, didMount]) => {
          const { exit, enter } = config as ScrollSeekConfiguration
          console.log("DID MOUNT", didMount);
          if (isSeeking) {
            if (exit(speed, range)) {
              return false
            }
 // ........
`

**Comment by petyosi** [comment]
True. It might have to wait for this flag: https://github.com/petyosi/react-virtuoso/blob/master/src/initialTopMostItemIndexSystem.ts#L10

**Comment by eliasws** [comment]
scrolledToInitialItem has the same problem :) (always true). 

Seems like 
https://github.com/petyosi/react-virtuoso/blob/392b9c8e717f9a937573c3db0a17d50d0d4a3132/src/initialTopMostItemIndexSystem.ts#L34 
is firing to fast.

For now i just disabled the scrollSeek.

**Comment by petyosi** [comment]
This flag will turn true after the initial destination has been reached. My understanding is that you are trying to avoid that call. 

At any case, I will look at that at some point.

**Comment by eliasws** [comment]
@petyosi FYI, just checked: Same happens with endless scroll ("prepending-items"); the velocity "jumps" for a moment

**Comment by petyosi** [comment]
That's understandable, based on how velocity and prepending work. Scrolling is readjusted to accommodate the prepended items.

**Comment by phenry20** [comment]
Any updates?

**Comment by petyosi** [comment]
@tomholford you can hook up to the `scrollerRef`, listen to scrollTop changes and calculate the velocity.

**Comment by tomholford** [comment]
> @tomholford you can hook up to the `scrollerRef`, listen to scrollTop changes and calculate the velocity.

Great suggestion, will give this a shot. Thanks!
